velox/docs/functions/spark/math.rst

```diff
@@ -240,10 +240,10 @@ Mathematical Functions
     If ``x`` contains non-hexadecimal character, the function returns NULL.
     When ``x`` contains even number of characters, each pair is converted to a single byte. The number of bytes in the result is half the number of bytes in the input.
     When ``x`` contains a single character, the result contains a single byte whose value matches the hexadecimal character.
-    When ``x`` contains an odd number characters greater than 2, the first character is ignored, the remaining pairs of characters are converted to bytes, then zero byte is added at the end of the output. ::
+    When ``x`` contains an odd number characters greater than 2, the first character is decoded into the first byte of the result and the remaining pairs of characters are decoded into subsequent bytes. This behavior matches Spark 3.3.2 and newer. ::
```

Comment:
I just realized that we no longer need to describe the behavior separately for 1-char input and 3+ odd chars inputs. These are not consistent. Hence, we can remove "When ``x`` contains a single character..." sentence and remove "greater than 2" from "When ``x`` contains an odd number characters greater than". Also, let's add missing "of" to "an odd number of characters". Thanks.

Reply:
fixed, thank you!